modules/django-social-auth/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class SocialAuthConfig(AppConfig):
+    name = "modules.social-auth"
```

Comment:
It's actually a valid python path:
```shell
yarn run demo
yarn run add django-social-auth
```
```shell
python manage.py show_urls | grep "modules."
# => /modules/social-auth/apple/connect/     modules.social-auth.views.AppleLogin    social_apple_connect
# => /modules/social-auth/apple/login/       modules.social-auth.views.AppleLogin    social_apple_login
# => /modules/social-auth/facebook/connect/  modules.social-auth.views.FacebookLogin social_facebook_connect
# => /modules/social-auth/facebook/login/    modules.social-auth.views.FacebookLogin social_facebook_login
# => /modules/social-auth/google/connect/    modules.social-auth.views.GoogleLogin   social_google_connect
# => /modules/social-auth/google/login/      modules.social-auth.views.GoogleLogin   social_google_login
# => /modules/social-auth/socialaccounts/    rest_auth.registration.views.SocialAccountListView      social_account_list
# => /modules/social-auth/socialaccounts/<int:pk>/disconnect/        rest_auth.registration.views.SocialAccountDisconnectView      social_account_disconnect
```

The reason why I named all `apps.py` with `-` instead of `_` was because we are adding modules dynamically into `INSTALLED_APPS`, and the name of the modules is derived from the module directory path:
https://github.com/crowdbotics/django-scaffold/blob/ecff5e4334d864e9d21db89e2eeae9b109dc39f9/%7B%7Bcookiecutter.project_slug%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/settings.py#L91
https://github.com/crowdbotics/django-scaffold/blob/ecff5e4334d864e9d21db89e2eeae9b109dc39f9/%7B%7Bcookiecutter.project_slug%7D%7D/modules/manifest.py#L14-L15

In other words, naming all `AppConfig.name` with dashes was intentional, so that it follows the registered name in `INSTALLED_APPS`.

Reply:
To my surprise it is legal, but you have to resort to this to use it from others part of the code:
https://stackoverflow.com/a/37831973

Reply:
I've updated all django modules to use underscores

Reply:
@danielsousaio it's good, but why don't we focus on PLAT-7015? It has all the refactor to python modules?